src/app/downloads/downloads.component.ts

```diff
@@ -2,97 +2,81 @@ import { Component, OnInit, OnDestroy } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { Subscription } from 'rxjs/Subscription';
 import { CastleService } from '../core';
-import { EpisodeModel, EpisodeMetricsModel, PodcastMetricsModel, INTERVAL_DAILY, FilterModel } from '../ngrx/model';
-import { castlePodcastMetrics, castleEpisodeMetrics, castleFilter } from '../ngrx/actions/castle.action.creator';
+import { EpisodeModel, INTERVAL_DAILY, FilterModel } from '../ngrx/model';
+import { CastleFilterAction, CastlePodcastMetricsAction, CastleEpisodeMetricsAction } from '../ngrx/actions';
+import { selectFilter, selectEpisodes, filterAllPodcastEpisodes } from '../ngrx/reducers/reducers';
 
 @Component({
   selector: 'metrics-downloads',
   template: `
-    <prx-spinner *ngIf="isPodcastLoading || isEpisodeLoading"></prx-spinner>
-    <div class="controls">
-      <metrics-canned-range></metrics-canned-range>
+    <section class="controls">
+      <metrics-interval></metrics-interval>
```

Comment:
yes, these components just look at the filter state and send out filter actions. We'll probably have some "dumb" components that take Input()s but all the components now are responsible for a part of the state, so that's how I've hooked them up.